tee-worker/README.md

```diff
@@ -42,6 +42,22 @@ make
 ### Integration Tests
 See [docker/README.md](docker/README.md)
 
+### Quickly launch parachain, tee worker and run ts-test against tee-worker
+
+```
+source "$HOME/.cargo/env"
```

Comment:
you only need to run it once after rustup installation so I think it is not necessary to include it here.

Reply:
added it as comment in case new beginner doesn't have it in bash profile.